src/services/personalities/sustain_focus.py

```
@@ -0,0 +1,50 @@
+"""
+SustainFocus personality: Specialist in sustainability and long-term thinking.
+"""
+from typing import List
```

Ruff (0.8.2)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)